pkg/vault/contracts/BaseHooks.sol

```diff
@@ -8,6 +8,7 @@ import { IBasePool } from "@balancer-labs/v3-interfaces/contracts/vault/IBasePoo
 import {
     AddLiquidityKind,
     HooksConfig,
+    HookFlags,
     LiquidityManagement,
     RemoveLiquidityKind,
     TokenConfig
```

Comment:
I thought about removing the VaultGuard from BaseHooks since it won't use the `onlyVault` modifier, but maybe it's better to keep it so external developers have a more standard way to use the vault in hooks.

Reply:
Yes; I also think it's better to leave in at this layer, even though it's not directly used, as otherwise the derived contracts would have to use it. I can add a comment about it, though.